testing/dart_isolate_runner.cc

```diff
@@ -124,6 +122,15 @@ std::unique_ptr<AutoIsolateShutdown> RunDartCodeInIsolateOnUITaskRunner(
   context.advisory_script_uri = "main.dart";
   context.advisory_script_entrypoint = entrypoint.c_str();
 
+  std::vector<std::string> entrypoint_args;
+#if !OS_FUCHSIA
+  entrypoint_args = args;
+#else
```

Comment:
Ditto.

Reply:
I'm not sure how to change it here

Reply:
But I am sure that the logic here has not changed in Fuchsia.

Reply:
See above for how to change this

Reply:
Thanks! Done.